src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -137,6 +139,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ annotatorState: prevAnnotatorState }: Props): void {
+        const { error: prevError } = prevAnnotatorState;
+        const {
+            annotatorState: { error },
+        } = this.props;
+
+        if (error && prevError !== error) {
+            switch (error.code) {
+                case ERROR_CODE_FETCH_ANNOTATIONS:
+                    this.removeAnnotations();
+                    break;
+                default:
+                // noop for unknown error code
+            }
+        }
+    }
+
+    removeAnnotations = (): void => {
+        const { feedItems } = this.state;
+
+        if (!feedItems) {
+            return;
+        }
+
+        this.setState({
```

Comment:
Rather than directly manipularting the local state, should we update the feedAPI cache, instead? We could add a new `clearFeedItems` method that accepts a type parameter, then refresh the feed. Otherwise, the local state may be restored from the "stale" cache at a later time.